src/object/obj_rpc.h

```diff
@@ -540,13 +583,43 @@ enum daos_cpd_sg_type {
 };
 
 /** Scatter/gather info for CPD RPC data structure. */
+
+/* For 2.0 or older */
+struct daos_cpd_sg_20 {
```

Comment:
why do you need a 2.0 struct here if it's the same as the 2.2 one?

Reply:
We need to trigger different cpr_proc functions, here is the entry point, please check crt_proc_struct_daos_cpd_sg_20()/crt_proc_struct_daos_cpd_sg() for detail.